app/repositories/PublishedRepository.scala

```diff
@@ -55,7 +55,7 @@ class PublishedRepositoryImpl @Inject() (mongoComponent: ReactiveMongoComponent)
       "$set" -> Json.obj(
         "process" -> process,
         "publishedBy" -> user,
-        "datePublished" -> Json.obj("$date" -> LocalDateTime.now.atZone(ZoneId.of("UTC")).toInstant.toEpochMilli)
+        "datePublished" -> Json.obj("$date" -> ZonedDateTime.now.toInstant.toEpochMilli)
```

Comment:
There is a discrepancy here with the viewer. The viewer uses Instant.now to capture the time of submission etc. whereas here we are using ZonedDateTime.now. I know they are equivalent.

Reply:
In the viewer the Instant.now (in UTC) is used to let Mongo know (in UTC) how long it is since the document was last accessed. This data is exclusively for use by Mongo to decide when it should delete the document. Instant is always in UTC, so is ideal to determine the UTC datatime in milliseconds. The datetime above is an application datetime exclusively for use by the application and is Zoned in common with all other application datetimes.

Reply:
When we are saving an update date/time created dynamically in the repository to Mongo it doesn't really matter if we use Instant.now.toEpochMilli or ZonedDateTime.now.toInstant.toEpochMilli as they are equivalent. What is important is that the reads method converts UTC milliseconds into ZonedDateTime with the correct zone as that is how dates are represented in the application (which it does).